examples/log-generator/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8.2.1-alpine
+
+WORKDIR /log-generator
+
+ENV ESCRIBA_TIMEOUT 3000
```

Comment:
I did some tests here and you are right. I ended up confusing ENV with ARG.   I thought that ENV are only used in buildtime and ARG in runtime but it’s the opposite.